File: Clases_Csharp/Cita.cs
namespace SistemaVeterinaria
{
    public class Cita
    {
        public string IdCita { get; private set; }
        public string Fecha { get; private set; }
        public string Hora { get; private set; }
        public Mascota Mascota { get; private set; }
        public Veterinario Veterinario { get; private set; }
        public bool Ocupado { get; private set; }

        public Cita(string idCita, string fecha, string hora, Mascota mascota, Veterinario veterinario)
        {
            IdCita = idCita;
            Fecha = fecha;
            Hora = hora;
            Mascota = mascota;
            Veterinario = veterinario;
            Ocupado = true;
        }
    }
}

File: Clases-Bluej/Sistema_ticketing/Cita.java
public class Cita {
    private String idCita;
    private String fecha;
    private String hora;
    private Mascota mascota;
    private Veterinario veterinario;
    private boolean ocupado;

    public Cita(String idCita, String fecha, String hora, Mascota mascota, Veterinario veterinario) {
        this.idCita = idCita;
        this.fecha = fecha;
        this.hora = hora;
        this.mascota = mascota;
        this.veterinario = veterinario;
        this.ocupado = true; // Indica que el horario está reservado
    }

    public String getIdCita() { return idCita; }
    public String getFecha() { return fecha; }
    public String getHora() { return hora; }
    public Mascota getMascota() { return mascota; }
    public Veterinario getVeterinario() { return veterinario; }
    public boolean isOcupado() { return ocupado; }
}